### PR TITLE
Project Section

### LINKED ISSUE #81: CODING: Develop the projects section on projects section
### Description
As a developer,
I need the following section of mockup to be developed,
so that I can add it in my projects page :  https://xd.adobe.com/view/e86737c2-ff84-4e23-b4b1-e541508d5279-bd34/

### mocks

spacing and all details

image

image

image

{font style-avenir color style-#103B81 Weight-medium}
### Acceptance Criteria

- [x] Follows all the design guidelines as mentioned and is responsive enough on large screen devices.
- [x] when clicked on ( view more ) all the listed projects should up .
- [x] the github and zulip redirect links should work in all  cards .

### Definition of Done

- [x]  All of the required items are completed.
- [x]  Approval by 2 mentor.

### Estimation
6 -7 hours

HAPPY CODING !! 😃

### PR DESCRIPTION
**Pull request by happycoder0011**
### Description

Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #81 

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "version": "0.1.2",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
```
Comment (annabauza) [inline package.json:7]: material-ui/core is not react native dependency it is rather react js dependency. We can try https://www.npmjs.com/package/react-native-material-ui

File: src/Components/Projects/MainSection/cards/index.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          <img src={zulip} alt="Zulip"/>
+            <p>Zulip repo</p>
+           </Box>
+          </Link>
+          <Typography className={classes.text}>
+          <Box fontSize={18} fontWeight="fontWeightMedium" fontFamily="avenir ">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate.
+          </Box>
+          </Typography>
+           <div className="tag-container">
```
Comment (annabauza) [inline src/Components/Projects/MainSection/cards/index.js:78]: we do not use divs it is not allowed in react native

Comment (annabauza) [inline src/Components/Projects/MainSection/cards/index.js:78]: we do not use divs it is not allowed in react native

File: src/Components/Projects/MainSection/cards/index.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
...WZjcWeh8Oaudh2ZLqbalP5oznf1VMotlzW8SB3mFu9y2draYHJR5V+K9Ia4bO4NwuaQRqPspSpc1J/zNCl2tau4M1DS/5Gd13V7mQ0/DuHBS4XKIS/u0SMt/ZNq7CVbTRgxNpmzKoySjCjCcYc0Wo4IkLUDtQyaDgsJt1PDUe3mt7v0zSd0WG7Q08NWeKtxp5zpGLgK64IkqqI5XEGlBAIrqC6UAEEESuYb4IAtDMkpYolJsBGQHCjsRCjsQD+56eKvSb+seBn6LY7PUcAFmOwlmx2xn6QXfT6rZW2MLn5fXTw6kM2Wop1ZqxOqVFjCWpUIUe1ujqzuTlrk3YyEoHJoSw5aUV7oSPvEk95KZmha1dMa1pKfCikatmS9m6RFqcXAjisr26sRYQY0d4FbQ2yKke027waU702G5S5yWMmnJFQpnJBy6nIAcgioIAORUCjAIAIlfcOco6I75uxAGQBRUEB1HYk0dpQF49ltOa9R3BrR3n9lr7Cso9lo/4rv1NHgVpzai5s7/AGrr45/WHwKM0pmKqMKqTZ9Hkrhcm4qroKNjRUuQaiI7UA4YEaAk2uQ94mID1UNvWTRhWtz1Xtq2YqLui3bWBuGFzhzXHJe9mYax55pAronjkymqKguILWOwukopRXuyQBqZRTqUKKSDjJQCq4SuNeuEoAwKUak2pQIDQPZifhqf1j/aFoges/8AZu38Jx41D4AK9tK4+T/Ku7j/AMIch6OHJs0pUFIc5BR2uTbEjB6Bo5a5KhyaCojB62Vlh2KiK56bY0V1Rbtmi73qOvRuJhCWNREr5grNjTDdrKOGr2kKIa5W3b+zQ4nnKpzCuvC7jk5ZrIclBCVxOm6kqhyRwUjUKAWpaJFqVGibsQCs7kVozR2hFJzQCjUoCkgjgoDUPZ9TiztPEuP+qPori1VvZGjhs9IfoHjn9VYWuXFle69DGaxhYFGlI40bEkPCzUo0JBjkuCtZRwECuhccVpdiFy4XLjyknOWNKBy7iSONDEgKH7QLNIPRZkwrYNsqUsJWQ1mw4jmunivTl5p9GagitKCsg4zRJO1XUEAq/RJsQQQB0mgggFGrp0XUEBtNzCKbeQb5KRBzQQXBXpx1HaUEErSrSnTdEEE0LXWlFnVBBaUi8pOUEFjY44rkoILGoTaQfhnosdvEfiOQQXRwubn8ItQQQXQ5n//Z" />
```
Comment (annabauza) [inline src/Components/Projects/MainSection/cards/index.js:94]: we have assets folder for holding images

File: yarn.lock [hunk 4]
```diff
@@ -973,6 +973,13 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
+"@babel/runtime@^7.3.1", "@babel/runtime@^7.4.4", "@babel/runtime@^7.5.5", "@babel/runtime@^7.8.3", "@babel/runtime@^7.8.7":
```
Comment (annabauza) [inline yarn.lock:976]: why do you need updating yarn.lock?

**Comment by nandini45** [comment]
@happycoder0011  will u do the work?

**Comment by happycoder0011** [comment]
Ok I am working

**Pull request closed without merging**